lib/node_modules/@stdlib/math/base/special/fibonacci/benchmark/benchmark.js

```diff
@@ -45,10 +45,11 @@ bench( pkg, function benchmark( b ) {
 	var y;
 	var i;
 
+	x = discreteUniform( 100, 0, 79 );
```

Comment:
```suggestion
	x = discreteUniform( 100, 0, 78 );
```

randu generates values on the interval `[0, 1)`. Hence, the previous logic had a max `x` of `78`.